### DISCUSSION TITLE
pass direct references to webpack.config module.rules.loader rather than magic strings

### DISCUSSION THREAD

**Discussion by danehansen** [webpack/webpack#19017] [Q&A]
is there a way to pass in a direct reference for the loaders `ts-loader`, `css-loader`, and `sass-loader` into webpack.config so that i so that these loaders can be bundled in with the exported config file? kinda like how `mini-css-extract-plugin` has `MiniCssExtractPlugin.loader` to pass in... thanks!

**Comment by alexander-akait** [comment]
You can use `require.resolve("ts-loader")`
